### PR TITLE
fixed test warnings

### PR DESCRIPTION
**Pull request by matrosoe**
Fixed test warnings in aggregate node tests
Will work on other warnings

### CODE REVIEW

**Comment by jetuk** [comment]
@matrosoe is this ready now?

**Comment by matrosoe** [comment]
Think so!

**Comment by jetuk** [comment]
I can't merge it until you remove the draft tag.

File: tests/test_parameters.py [hunk 1]
```diff
@@ -67,9 +67,6 @@ def test_being_a_variable(self, simple_linear_model):
 
         np.testing.assert_allclose(p.get_double_lower_bounds(), np.array([np.pi/2]))
         np.testing.assert_allclose(p.get_double_upper_bounds(), np.array([2*np.pi]))
-        # Test deprecated API too.
-        np.testing.assert_allclose(p.lower_bounds(), np.array([np.pi/2]))
```
Comment (jetuk) [inline tests/test_parameters.py]: I think we should probably remove the actual API functions on the `Parameter` object too.

**Pull request merged**